### PR TITLE
Closes #1863 Update lab grading vignette

### LINKED ISSUE #1863: Documentation:  Update Lab grading Vignette for NCICTCAEv5
### Please select a category the issue is focused on?

User Guides

### Let us know where something needs a refresh or put your idea here!

- [x]  Add some more information about BASELINE value being abnormal for tests ALT, AST, ALKPH, BILI and GGT.
- [x]  Update introduction to Lab Grading Vignette around location of tox grading data and how to load into R. 
- [x]  Move Metadata and Example datasets closer together on Reference page @millerg23  do you mind doing this as well?

### PR DESCRIPTION
**Pull request by millerg23**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiraldev/devel/articles/development_process.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/devel/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/devel/articles/programming_strategy.html#deprecation)?
- [x] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/devel/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiral/cran-release/reference/index.html)" page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately 
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

**Comment by millerg23** [comment]
@bms63 @zdz2101 any ideas why checks are failing? Been a while since I did a pull request, so maybe I have missed something.
Once I get checks fixed I can invite others to QC. If you could give me some initial feedback that would be great.

**Comment by bundfussr** [comment]
> @bms63 @zdz2101 any ideas why checks are failing? Been a while since I did a pull request, so maybe I have missed something. Once I get checks fixed I can invite others to QC. If you could give me some initial feedback that would be great.

I think it is an issue on github. See https://www.githubstatus.com/.

File: docs/pkgdown.yml [hunk 1]
```diff
@@ -19,7 +19,8 @@ articles:
   queries_dataset: queries_dataset.html
   questionnaires: questionnaires.html
   visits_periods: visits_periods.html
-last_built: 2023-03-02T20:56Z
+last_built: 2023-05-11T13:11Z
 urls:
-  reference: https://pharmaverse.github.io/admiral/cran-release/reference
-  article: https://pharmaverse.github.io/admiral/cran-release/articles
+  reference: https://pharmaverse.github.io/admiral/reference
```
Comment (zdz2101) [inline docs/pkgdown.yml:24]:
```suggestion
  reference: https://pharmaverse.github.io/admiral/cran-release/reference
```

File: docs/pkgdown.yml [hunk 2]
```diff
@@ -19,7 +19,8 @@ articles:
...
+  article: https://pharmaverse.github.io/admiral/articles
```
Comment (zdz2101) [inline docs/pkgdown.yml:25]:
```suggestion
  article: https://pharmaverse.github.io/admiral/cran-release/articles
```

**Review by zdz2101 — COMMENTED** [review]
I think this modification you made is what's breaking the links check

**Review by bms63 — APPROVED** [review]
Looks good to me!!

**Comment by millerg23** [comment]
Should we add anyone else to this review, do we know if Paulina has a GitHub account?

**Review by zdz2101 — APPROVED** [review]
Looks good! Per lab-wg meeting, add news update?

**Comment by millerg23** [comment]
Updated NEWS.md file.

**Review by zdz2101 — APPROVED** [review]

**Pull request merged**